packages/firebase_ml_vision/lib/src/barcode_detector.dart

```diff
@@ -4,18 +4,599 @@
 
 part of firebase_ml_vision;
 
+/// Detector for performing optical character recognition(OCR) on an input image.
```

Comment:
I believe this was copied from the TextDetector. Can you update this for the barcode detector?

Reply:
😱